lnwallet/wallet.go

```diff
@@ -413,9 +417,10 @@ out:
 // commitment transaction is valid.
 func (l *LightningWallet) InitChannelReservation(
 	capacity, ourFundAmt btcutil.Amount, pushMSat lnwire.MilliSatoshi,
-	commitFeePerKw SatPerKWeight, fundingFeePerVSize SatPerVByte,
-	theirID *btcec.PublicKey, theirAddr net.Addr,
-	chainHash *chainhash.Hash, flags lnwire.FundingFlag) (*ChannelReservation, error) {
+	minConfs uint32, commitFeePerKw SatPerKWeight,
```

Comment:
Maybe export the `InitFundingReserveMsg` (or create an alias) to clean up the number of arguments to this method.

Reply:
Fixed.